<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSliderDemoClass</class>
 <widget class="QDialog" name="QSliderDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSliderDemo</string>
  </property>
  <widget class="QSlider" name="hsRed">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>30</y>
     <width>160</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>75</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
  </widget>
  <widget class="QSlider" name="hsGreen">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>160</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>67</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
  </widget>
  <widget class="QSlider" name="hsBlue">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>160</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>34</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
  </widget>
  <widget class="QSlider" name="hsAlpha">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>160</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBelow</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Alpha</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDial" name="diContorl">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>181</width>
     <height>131</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="pageStep">
    <number>100</number>
   </property>
   <property name="value">
    <number>456</number>
   </property>
   <property name="invertedAppearance">
    <bool>false</bool>
   </property>
   <property name="invertedControls">
    <bool>false</bool>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="notchesVisible">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNum">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QSliderDemo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>hsAlpha</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QSliderDemoClass</receiver>
   <slot>on_color_select()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hsBlue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QSliderDemoClass</receiver>
   <slot>on_color_select()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hsGreen</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QSliderDemoClass</receiver>
   <slot>on_color_select()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hsRed</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QSliderDemoClass</receiver>
   <slot>on_color_select()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>diContorl</sender>
   <signal>valueChanged(int)</signal>
   <receiver>QSliderDemoClass</receiver>
   <slot>on_contorl_select()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_color_select()</slot>
  <slot>on_contorl_select()</slot>
 </slots>
</ui>
